### DISCUSSION TITLE
Tailwind.config.js possible error in content

### DISCUSSION THREAD

**Discussion by andrewberryca** [tailwindlabs/tailwindcss#8876] [General]
The config stops working when there are spaces between file extensions:

"./src/**/*.{js,jsx,ts,tsx}" - working
"./src/**/*.{js, jsx, ts, tsx}" - not working

Should spaces be ignored so either of string will be correct?

**Comment by tofuwu-dev** [comment]
Paths are configured as [glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)), making it easy to match all of the content files in your project without a ton of configuration
```bash
module.exports = {
  content: [
    './src/**/*.{js,jsx,ts,tsx}',
  ],
  // ...
}
```
then you will be able to apply tailwind utility class in all js, jsx, ts, tsx file...
.{js,jsx,ts,tsx} => `*.js,|*.jsx,|*.ts,|*.tsx`
When you use `"./src//*.{js, jsx, ts, tsx}"` including whitespace, it will be not working as you find it...
It will be .js, .` `jsx,  .` `ts,  .` `tsx
Actually, you wont have such files, so it wont work properly, I guess
